CHANGES.md

```diff
@@ -13,6 +13,7 @@ Change Log
 * Fixed globe materials when `Globe.enableLighting` was `false`. [#6042](https://github.com/AnalyticalGraphicsInc/cesium/issues/6042)
 * Fixed shader compilation failure on pick when globe materials were enabled. [#6039](https://github.com/AnalyticalGraphicsInc/cesium/issues/6039)
 * Fixed crash when `invertClassification` was enabled, the invert color had an alpha less than `1.0`, and the window was resized. [#6046](https://github.com/AnalyticalGraphicsInc/cesium/issues/6046)
+* Added ability to create partial ellipsoid volumes using both the Entity API and CZML. New ellipsoid geometry properties: innerRadii, minimumAzimuth, maximumAzimuth, minimumElevation, maximumElevation. See the updated [Sandcastle example](https://cesiumjs.org/Cesium/Apps/Sandcastle/?src=Spheres%20and%20Ellipsoids.html&label=Geometries). [#5995](https://github.com/AnalyticalGraphicsInc/cesium/pull/5995)
```

Comment:
Perhaps move this to a new 1.42 section (release date: 2018-02-01); I do not know that we will have enough time to review this for 1.41 given the holidays.